timescale-docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM timescale/timescaledb-ha:pg14.3-ts2.7.0-p0
```

Comment:
Please provide documentation of what the intent of this odd looking dockerfile is.